bBridgeAPISDK/ImageProcessing/Structs/ObjectDetectionData.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace bBridgeAPISDK.ImageProcessing.Structs
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class ObjectDetectionData
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Create imput structure for Image Object Detection
+        /// </summary>
+        /// <param name="imageURL">Url of the image to detect from</param>
+        /// <param name="threshold">Object detection threshold 0 - 1 </param>
+        public ObjectDetectionData(string imageURL, double threshold)
+        {
+            if (!Uri.IsWellFormedUriString(imageURL, UriKind.Absolute))
+            {
+                throw new ArgumentException("Image uri is not in a valid format");
+            }
+
+			if (threshold < 0 || threshold > 1)
+            {
+                throw new ArgumentException("Image detection treshold must be with the range 0.01 - 1");
```

Comment:
The message of exception is incorrect. Now it is `"Image detection threshold must be with the range 0.01 - 1"` but it must be `"Image detection threshold must be with the range 0 - 1"` because `0 <= threshold <= 1`